### PR TITLE
chore: lint spec-main

### PR DESCRIPTION
**Pull request by nornagon**
#### Description of Change
We haven't been running eslint over spec-main. Now we are.

#### Checklist

- [ ] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [ ] relevant documentation is changed or added
- [ ] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [ ] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: none

### CODE REVIEW

**Comment by codebytere** [comment]
@nornagon this needs a rebase but it looks fine otherwise so i'll 👍 when it's green!

File: .eslintrc.json [hunk 1]
```diff
@@ -17,18 +17,30 @@
...
     }],
+    "standard/no-callback-literal": "off",
     "node/no-deprecated-api": 0
   },
   "parserOptions": {
     "ecmaVersion": 6,
     "sourceType": "module"
   },
+  "globals": {
+    "isCI": "readonly",
```
Comment (codebytere) [inline .eslintrc.json]: did we remove this?

Comment (nornagon) [reply .eslintrc.json]: yep, this PR came before that got merged. will fix

**Comment by nornagon** [comment]
Thanks @jkleinsc !

**Pull request merged**